File: Selenium/Project/Activity2.java
import io.github.bonigarcia.wdm.WebDriverManager;
import org.junit.jupiter.api.Assertions;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.openqa.selenium.support.ui.WebDriverWait;
import org.testng.annotations.AfterClass;
import org.testng.annotations.BeforeClass;
import org.testng.annotations.Test;

import java.time.Duration;

public class Activity2 {
    WebDriver driver;
    WebDriverWait wait;

    @BeforeClass
    public void setup() {
        WebDriverManager.firefoxdriver().setup();
        driver = new FirefoxDriver();
        driver.get("https://alchemy.hguy.co/jobs");
        wait = new WebDriverWait(driver, Duration.ofSeconds(10));
    }

    @Test
    public void verifyHeader() {
        WebElement header = driver.findElement(By.className("entry-title"));
        String headerText = header.getText();
        String expectedHeader="Welcome to Alchemy Jobs";
        Assertions.assertEquals(expectedHeader, headerText);
    }

    @AfterClass
    public void cleanup() {
        driver.quit();
    }
}

File: Selenium/Project/Activity2.py
# Import required packages from selenium
import pytest
from selenium.webdriver.common.by import By


class TestClass():

    def test_header_text(self, my_fixture):
        driver = my_fixture
        url = 'https://alchemy.hguy.co/jobs'
        driver.get(url)
        expected_header = 'Welcome to Alchemy Jobs'
        actual_header = driver.find_element(By.CLASS_NAME, "entry-title").text
        assert actual_header == expected_header
